### DISCUSSION TITLE
快速开始 | Hexo Theme Nexmoe

### DISCUSSION THREAD

**Discussion by giscus** [theme-nexmoe/hexo-docs#22] [General]
# 快速开始 | Hexo Theme Nexmoe

你也可以用本项目的示例快速起步 Nexmoe 主题 Hexo 示例，帮助你快速起步

https://docs.nexmoe.com/start/?giscus=3d536676740cdb02cf0ee668PxCR9vRX%2FHAD3%2BmnVXUZNXvT2DC0LyQnSOFYDgPeKiiBc%2BzejqVlVew4R2DoD2zs%2F9iAbdOP%2FZYFMI5WQkkDldxiSkbvNjYckHL8hoknegW3fQoyN7JhiyvqvCg%3D

**Comment by ice-deep-dream** [comment]
Nice
